packages/cli/lib/pipeline.js

```diff
@@ -12,10 +12,11 @@ const discoverOperations = async () => {
   for await (const { manifest } of discoverManifests()) {
     manifest
       .has(rdf.ns.rdf.type, ns.p.Operation)
-      .forEach(x => { // distinguish EcmaScript and EcmaScriptModule?
-        const link = x.out(ns.code.implementedBy).out(ns.code.link).term
-
-        ops.set(x.term, link)
+      .forEach(x => {
+        const impl = x.out(ns.code.implementedBy)
+        const type = impl.out(ns.rdf.type).term
+        const link = impl.out(ns.code.link).term
+        ops.set(x.term, { type, link })
```

Comment:
Maybe `x` is not greatest variable name :D
```suggestion
      .forEach(operation => {
        const impl = operation.out(ns.code.implementedBy)
        const type = impl.out(ns.rdf.type).term
        const link = impl.out(ns.code.link).term
        ops.set(operation.term, { type, link })
```